cassandane/Cassandane/Unit/TestCase.pm

```diff
@@ -329,4 +330,12 @@ sub assert_num_lte
                   "$actual is not less-than-or-equal-to $expected");
 }
 
+sub new_test_url {
+  my ($self, $content_or_app) = @_;
+
+  return Cassandane::Util::TestURL->new({
+    app => $content_or_app,
+  });
+}
+
```

Comment:
Four space indents in cyrus ;)

Reply:
Yeah whoops! I thought I had fixed everywhere but missed this one, thanks

Reply:
Fixed